libraries/core-react/stories/docs/IconPreview.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import { Story, Meta } from '@storybook/react'
+import styled from 'styled-components'
+import { Icon, Button, Typography, TableOfContents } from '../../src'
+import fileDownload from 'js-file-download'
+import systemIcons from '../assets/icons/system-icons.json'
```

Comment:
We need to update the `figma-broker` to save the json file at this location.

Reply:
In this pr or later? Need help to update `figma-broker` then

Reply:
You only need to update two lines.
* Change this [line](https://github.com/equinor/design-system/blob/384947945295befc741752eed4b90f7dd44020be/apps/figma-broker/constants.js#L15) to wanted path.
* Uncomment this [line](https://github.com/equinor/design-system/blob/384947945295befc741752eed4b90f7dd44020be/apps/figma-broker/actions/createAssets.js#L164).